pages/aws.md

```diff
@@ -38,20 +22,11 @@ Your username will be created for you during onboarding. Check the MFA Token box
 
 ### Servers
 
-The core service of AWS is the [Elastic Compute Cloud](http://aws.amazon.com/documentation/ec2/) (EC2).
-These are virtual machines just like on your computer, but hosted in the AWS
-environment. You receive access through a GUI, the [AWS Management Console](https://18f.signin.aws.com),
-an [API](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/query-apis.html),
-and to the guest [OS through SSH](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html).
+The core service of AWS is the [Elastic Compute Cloud](http://aws.amazon.com/documentation/ec2/) (EC2). These are virtual machines just like on your computer, but hosted in the AWS environment. You receive access through a GUI, the [AWS Management Console](https://18f.signin.aws.com), an [API](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/query-apis.html), and to the guest [OS through SSH](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html).
```

Comment:
https://18f.signin.aws.com doesn't resolve. Should this be https://18f.signin.aws.amazon.com/console instead?